commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/ftp/FtpProviderTestCase.java

```diff
@@ -113,6 +114,14 @@ static void setUpClass(final String rootDirectory, final FileSystemFactory fileS
         if (commandFactory != null) {
             serverFactory.setCommandFactory(commandFactory);
         }
+        ConnectionConfigFactory configFactory = new ConnectionConfigFactory();
```

Comment:
Follow the code convention in the file you are editing: In this case, use `final`.